### ISSUE TITLE
Side toolbar error: Uncaught TypeError: _this.props.store.getItem(...) is not a function

### ISSUE THREAD

**Issue by charlex** [draft-js-plugins/draft-js-plugins#770]
Toolbar isn't loading. And I get this error every keypress.

Error:
```Uncaught TypeError: _this.props.store.getItem(...) is not a function```

Location:
https://github.com/draft-js-plugins/draft-js-plugins/blob/master/draft-js-side-toolbar-plugin/src/components/Toolbar/index.js#L40

This is my source code:

```
import React, { Component } from 'react';
import Editor, { createEditorStateWithText } from 'draft-js-plugins-editor'; // eslint-disable-line import/no-unresolved
import createSideToolbarPlugin from 'draft-js-side-toolbar-plugin'; // eslint-disable-line import/no-unresolved
import 'draft-js-inline-toolbar-plugin/lib/plugin.css';
import 'draft-js-side-toolbar-plugin/lib/plugin.css';


// Sidebar toolbar
const sideToolbarPlugin = createSideToolbarPlugin();
const { SideToolbar } = sideToolbarPlugin;

// Plugins
const plugins = [
  sideToolbarPlugin,
];
const text = 'In this editor a toolbar shows up once you select part of the text …';

export default class CardEditor extends Component {

  state = {
    editorState: createEditorStateWithText(text),
  };

  onChange = (editorState) => {
    this.setState({
      editorState,
    });
  };

  focus = () => {
    this.editor.focus();
  };

  render() {
    return (
      <div onClick={this.focus}>
        <Editor
          editorState={this.state.editorState}
          onChange={this.onChange}
          plugins={plugins}
          ref={(element) => { this.editor = element; }}
        />
        <SideToolbar />
      </div>
    );
  }
}
```

**Comment by lineldcosta** [comment]
Even i am facing the same issue!

**Comment by patleeman** [comment]
I had the same issue, then the problem went away after reinstalling draft-js-plugins-editor:

```
npm install draft-js-plugins-editor@beta --save
```

[As per this page](https://www.draft-js-plugins.com/plugin/side-toolbar)

**Comment by deesx** [comment]
it seems that the error persists; or came back (recent react16 updates ?)

**Comment by troelskn** [comment]
Installing from beta doesn't do it for me either.

**Comment by troelskn** [comment]
I found the error. I had code where I would call `createSideToolbarPlugin()` inside the `render()` function of my component. This means a new plugin would be crated all the time. Moving `const sideToolbarPlugin = createSideToolbarPlugin();` to the global scope solved this.

**Comment by yhjor** [comment]
If we are using multiple editors, it is required to create multiple plugin... https://github.com/draft-js-plugins/draft-js-plugins/blob/master/FAQ.md#can-i-use-the-same-plugin-for-multiple-plugin-editors

My workaround would be creating a class property in the editor, so it would initialize only once per component: ```js
constructor(props) {
    super(props)
    const { plugins, InlineToolbar } = createEditorPlugins()
    this.plugins = plugins
    this.InlineToolbar = InlineToolbar
  }
```

**Comment by someshfengde** [comment]
same issue is happning again but now the error message is  _this.props.plugins.filter is not a function

**Comment by fxOne** [comment]
@someshfengde can you provide more information? The only place I could find is where `props.plugins.filter` is used is https://github.com/draft-js-plugins/draft-js-plugins/blob/780d732a1224dce3cb0918d0a00a57f303784c6c/packages/editor/src/Editor/index.tsx#L231 which is in the Editor itself.

**Comment by someshfengde** [comment]
wait let me send you some part of code :
I am trying to have draft mentions as well as draft emoji's
*https://pastebin.com/vgJ0j9Hi <https://pastebin.com/vgJ0j9Hi> *
--pastebin link
this is the code
thanks for responding quickly

**Comment by fxOne** [comment]
On a quick look I can see that you are creating the emoji plugin on every rendering. You are also not using the side toolbar. If you still have this issue can you please create a new issue and maybe provide a https://codesandbox.io example.

**Comment by sahilpurav** [comment]
I faced the similar issue with InlineToolbarPlugin and this plugin too. I was using draft.js in Next.js project. The problem was "Fast Refresh" feature of Next.js. I just added `// @refresh reset` this line at the start of my editor component. It has forced the next app to do full refresh only for the pages where my Editor is present.
